RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -38,6 +38,16 @@ import RevenueCat
     private(set) var appIsLatestVersion: Bool = defaultAppIsLatestVersion
     private(set) var purchasesProvider: CustomerCenterPurchasesType
 
+    var shouldShowAppUpdateWarning: Bool {
+        return !appIsLatestVersion && (configuration?.support.shouldWarnCustomerToUpdate ?? false)
+    }
+
+    /// Whether or not the user needs to update their app version to contact support.
+    var appUpdateRequiredToContactSupport: Bool {
+        // For now, we're using the same flag as the app update warning.
```

Comment:
I'd remove the "for now", since it assumes some kind of temporal nature to it, and if we never change it later on it will look to a maintainer like this was temporary or hacky code